### DISCUSSION TITLE
Sergen Error

### DISCUSSION THREAD

**Discussion by gfo2007** [serenity-is/Serenity#6534] [Help] [answered]
Hi,
When I try to run sergen via powershell, the available connections are listed but I am not prompted for the connection I want to select.

Below is the image. 
![sergen error](https://user-images.githubusercontent.com/14317420/190370190-d3aaa884-ff54-433c-9cf2-ae247f7850eb.PNG)
Please help.

**Comment by VictorTomaili** [comment, marked as answer]
Use up and down arrow keys and press enter

**Comment by gfo2007** [comment]
Thank you
